FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MainSignInFragment.kt

```diff
@@ -529,11 +524,7 @@ class MainSignInFragment : BaseSingInFragment<FragmentMainSignInBinding>() {
               val domain = EmailUtil.getDomain(account)
               val idToken = googleSignInAccount?.idToken ?: return@let
               fesUrl = GeneralUtil.generateFesUrl(domain)
-              domainOrgRulesViewModel.fetchOrgRules(
-                account = account,
-                idToken = idToken,
-                fesUrl = fesUrl
-              )
+              domainOrgRulesViewModel.fetchOrgRules(idToken = idToken, fesUrl = fesUrl)
```

Comment:
`org rules` should be called `client configuration` in all usage. Should also be `ClientConfigurationViewModel.kt`, `fetchClientConfiguration` and so on.

Reply:
@tomholub Got it. I will create a new issue for that. 

About this PR. It seems after dropping `uuid`(https://github.com/FlowCrypt/flowcrypt-android/issues/1990), we didn't use `fes` URL to fetch org rules and used `https://flowcrypt.com/api/account/get` instead during the login process. Only the last version was affected(1.3.9 https://github.com/FlowCrypt/flowcrypt-android/issues/2094). I think we need to roll out a quick release.

https://github.com/FlowCrypt/flowcrypt-android/blob/34086f24645e280f9473d8c73c082141f01abaa8/FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/DomainOrgRulesViewModel.kt#L42

Reply:
Was 1.3.9 ever released to the enterprise prod? It looks like maybe it wasn't?

Reply:
I've modified the tests. This kind of test was not simple to reproduce. I had a problem with DNS on Semaphore CI. Spent around 4 days to find out a way. But it still looks not perfect.

> We have to do an analysis on this, find out exactly what the impact was, what to do to prevent similar issues in the future.

Agree. Need to discuss some things. I will write you after the release.

Reply:
> Was 1.3.9 ever released to the enterprise prod? It looks like maybe it wasn't?

It was released to the Airbus closed testing track.

Reply:
> I've modified the tests. This kind of test was not simple to reproduce. I had a problem with DNS on Semaphore CI. Spent around 4 days to find out a way. But it still looks not perfect.

On browser extension, we don't modify the DNS. We use email accounts like `user@localhsot:8000` and then we have a mock api (including mock FES) running at fes.localhost:8000 which still resolves to localhost without any DNS records. In case of Android it wouldn't be localhost, but it shouldn't be principally any different than Gmail API mock we've done recently.

Reply:
> > Was 1.3.9 ever released to the enterprise prod? It looks like maybe it wasn't?
> 
> It was released to the *** closed testing track.

As long as it didn't reach production, not too bad. Probably broke their evaluation setup, but that will be fixed with a new version. 

Be careful mentioning client names in public discussion. I deleted the comment.

Reply:
I use localhost:1212. For `fes` it will be `fes.localhost:1212.` My PC and my laptop resolve `fes.localhost:1212`. But Semaphore CI can't resolve `fes.localhost`, just `localhost`. It seems their version of Ubuntu doesn't hove some apps/settings. I've tried to fix it but failed. I had to modify the `hosts` file on the Android emulator directly.